src/Select/Select.story.tsx

```diff
@@ -43,8 +43,17 @@ const wrappingOptions = [
   },
 ];
 
-const SelectWithManyOptions = ({ multiselect, labelText }: SelectProps) => {
+const PCNList = [
+  { value: "2", label: "PCN2" },
+  { value: "4", label: "PCN4" },
+  { value: "1", label: "PCN1" },
+  { value: "9", label: "PCN9" },
+];
+
+const SelectWithManyOptions = ({ multiselect, labelText, ...props }: SelectProps) => {
+  const selectRef = useRef(null);
```

Comment:
Do we need this ref?

Reply:
This ref is not required. Was added in the development process. Will remove it.